<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWoVersionUpgradeTipDialog</class>
 <widget class="QDialog" name="QWoVersionUpgradeTipDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Upgrade content:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="desc">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnIgnore">
       <property name="toolTip">
        <string>Ignore this version</string>
       </property>
       <property name="text">
        <string>Ignore</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn7Days">
       <property name="toolTip">
        <string>Pop up again in seven days</string>
       </property>
       <property name="text">
        <string>7 days</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnTomorrow">
       <property name="toolTip">
        <string>Pop up again tomorrow</string>
       </property>
       <property name="text">
        <string>Tomorrow</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnUpgrade">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Upgrade</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
